config/routes.rb

```diff
@@ -112,6 +111,11 @@
     resources :induction_coordinators, only: %i[index edit update], path: "induction-coordinators"
   end
 
+  namespace :schools do
+    resource :dashboard, controller: :dashboard, only: :show, path: "/"
+    resource :choose_programme, controller: :choose_programme, only: %i[show create], path: "choose-programme"
```

Comment:
Do we need controller names if they match resource name?

Reply:
Singular vs plural confusion I think